### ISSUE TITLE
weird results

### ISSUE THREAD

**Issue by crazygirl1992** [KumapowerLIU/Rethinking-Inpainting-MEDFE#15]
hello，

i use my data to train the network, and the test results are weird,like
![图片](https://user-images.githubusercontent.com/26948839/99894445-84e5e780-2cbe-11eb-974b-52c827f995be.png)

it's fill the mask region with the weird pixels ,like noise.why this condition  happened? thank you!

**Comment by Young-Zzz** [comment]
Me too...
I don't know how to solve this problem.

**Comment by 543877815** [comment]
![1383](https://user-images.githubusercontent.com/25082467/120292762-277ac500-c2f7-11eb-8cd1-a4e9d2533f37.png)
My result using pretrain model of celebA....

**Comment by Young-Zzz** [comment]
> ![1383](https://user-images.githubusercontent.com/25082467/120292762-277ac500-c2f7-11eb-8cd1-a4e9d2533f37.png)
> My result using pretrain model of celebA....
I use the celeba-HQ dataset，but what is the mask file? Is the 'irregular_mask'?If I use the celeba dataset,and what is the mask file ?Can u reply me~

**Comment by Young-Zzz** [comment]
> ![1383](https://user-images.githubusercontent.com/25082467/120292762-277ac500-c2f7-11eb-8cd1-a4e9d2533f37.png)
> My result using pretrain model of celebA....

I use the celeba-HQ dataset，but what is the mask file? Is the 'irregular_mask'?If I use the celeba dataset,and what is the mask file ?Can u reply me~

**Comment by zzktb** [comment]
@543877815 Something similar to what I've got. I double checked I'm using the CelebA models.
![0](https://user-images.githubusercontent.com/23032621/127401544-cb305bbe-fd9d-44e4-96e9-729d267c660d.png)

**Comment by LC2978** [comment]
May I ask, I have trained with my own dataset, and the test results are similar to yours. Have you found the reason

**Comment by zzktb** [comment]
@LC2978 I am using the pre-trained model they provided. It seems that the network is not working normally. I have no idea what would be the reason.

**Comment by KleinXin** [comment]
Same weird problem

![image](https://user-images.githubusercontent.com/13084622/133826638-bec1f02c-57be-4624-b124-5312355d3a01.png)

**Comment by htyjers** [comment]
@crazygirl1992 Have you solved the problem？could you help me
